raiden_contracts/contracts/MonitoringService.sol

```diff
@@ -90,18 +90,27 @@ contract MonitoringService is Utils {
 
     /// @notice Deposit tokens used to reward MSs
     /// Can be called by anyone several times and on behalf of other accounts
-    /// @param beneficiary The account benefitting from the deposit
-    /// @param amount The amount of tokens to be depositted
-    function deposit(address beneficiary, uint amount) public {
-        require(amount > 0);
+    /// @param beneficiary The account benefiting from the deposit
+    /// @param total_deposit Idempotent function that sets the total_deposit of
+    //// tokens of the beneficiary
+    function deposit(address beneficiary, uint256 total_deposit) public
+    {
+        require(total_deposit != 0);
```

Comment:
Ah, ok. I understand what you mean. But I see it as: the scope is to make sure  `total_deposit  > 0`. Because Solidity is still a young language that can still behave in unwanted ways, I would not completely trust it especially if it doesn't really cost us much. Plus, this `> 0` pattern was agreed upon in a previous smart contracts review session for uRaiden. I would rather keep it like this for now.

Reply:
Changed it back to the way it was. I was referring to UINT(-1) above, but doesn't make a difference I think.